wiki/Tournaments/TWC/2015/ru.md

```diff
@@ -112,9 +108,9 @@ outdated: true
 4. Первые две команды в каждой группе проходят в следующий этап.
    1. В зависимости от количества команд, возможны изменения.
 
-5. Дальнейшие этапы проводятся по [системе с двойным утешением](https://ru.wikipedia.com/Олимпийская_система_с_выбыванием_после_двух_поражений): выигравшая команда переходит в следующий этап, проигравшая попадает в сетку проигравших.
+5. Дальнейшие этапы проводятся по [системе с двойным утешением](https://ru.wikipedia.com/Олимпийская_система_с_выбыванием_после_двух_поражений "Wikipedia"): выигравшая команда переходит в следующий этап, проигравшая попадает в сетку проигравших.
```

Comment:
this is a very odd link, I suspect there may be more
```suggestion
5. Дальнейшие этапы проводятся по [системе с двойным утешением](https://ru.wikipedia.org/wiki/Олимпийская_система_с_выбыванием_после_двух_поражений "Wikipedia"): выигравшая команда переходит в следующий этап, проигравшая попадает в сетку проигравших.
```

Reply:
looks like this was the only broken wikipedia link in the `/Tournaments/` directory